src/Facets/OFTWrapperFacet.sol

```diff
@@ -148,6 +158,12 @@ contract OFTWrapperFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
         doesNotContainDestinationCalls(_bridgeData)
         noNativeAsset(_bridgeData)
     {
+        _checkProxyOFTAddress(
```

Comment:
Could move this into _startBridge to not have code duplicate imho

Reply:
I think if we put it to the `_startBridge`, then it will waste gas for `swapAndBridge...`.
but I agree with you, since the above case is just for reverts.